$OpenBSD: patch-get_iplayer,v 1.32 2010/02/26 18:48:06 sthen Exp $

- use id3tag not id3v2
- switch default to rtmpdump
- increase max retries for flash modes, increasing the chance of
getting full downloads if you're not using SWFVerification
(see FILESDIR/options.sample for an example of how to use this).

--- get_iplayer.orig	Thu Feb 25 23:55:19 2010
+++ get_iplayer	Fri Feb 26 10:21:45 2010
@@ -330,7 +330,7 @@ mkpath $profile_dir if ! -d $profile_dir;
 
 
 # get list of additional user plugins and load plugin
-my $plugin_dir_system = '/usr/share/get_iplayer/plugins';
+my $plugin_dir_system = '${TRUEPREFIX}/share/get_iplayer/plugins';
 my $plugin_dir_user = "$profile_dir/plugins";
 for my $plugin_dir ( ( $plugin_dir_user, $plugin_dir_system ) ) {
 	if ( opendir( DIR, $plugin_dir ) ) {
@@ -452,6 +452,7 @@ if ( ! keys %plugin_files ) {
 	$opt->{pluginsupdate} = 1;
 }
 # Update this script if required
+$opt->{packagemanager}='disable';
 update_script() if $opt->{update} || $opt->{pluginsupdate};
 
 
@@ -658,12 +659,12 @@ sub init_search {
 	delete $binopts->{vlc};
 	push @{ $binopts->{vlc} }, '-vv' if $opt->{debug};
 
-	$bin->{id3v2}		= $opt->{id3v2} || 'id3v2';
+	$bin->{id3v2}		= $opt->{id3v2} || 'id3tag';
 	$bin->{atomicparsley}	= $opt->{atomicparsley} || 'AtomicParsley';
 
 	$bin->{tee}		= 'tee';
 
-	$bin->{flvstreamer}	= $opt->{flvstreamer} || $opt->{rtmpdump} || 'flvstreamer';
+	$bin->{flvstreamer}	= $opt->{flvstreamer} || $opt->{rtmpdump} || 'rtmpdump';
 	delete $binopts->{flvstreamer};
 	push @{ $binopts->{flvstreamer} }, ( '--timeout', 10 );
 	push @{ $binopts->{flvstreamer}	}, '--quiet' if $opt->{quiet};
@@ -4224,7 +4225,7 @@ sub tag_file {
 		my ( $id3_name, $id3_episode, $id3_desc, $id3_channel ) = ( $prog->{name}, $prog->{episode}, $prog->{desc}, $prog->{channel} );
 		s|"|\\"|g for ($id3_name, $id3_episode, $id3_desc, $id3_channel);
 		# Only tag if the required tool exists
-		if ( main::exists_in_path('id3v2') ) {
+		if ( main::exists_in_path('id3tag') ) {
 			main::logger "INFO: id3 tagging $prog->{ext} file\n";
 			my @cmd = (
 				$bin->{id3v2}, 
